manual_correction.py

```diff
@@ -87,31 +87,27 @@ def get_parser():
             """)
     )
     parser.add_argument(
-        '-path-in',
+        '-path-img',
         metavar="<folder>",
         required=True,
-        help='Path to the processed data. Example: ~/<your_dataset>/data_processed or ~/<your_dataset>/data_to_correct',
+        help='Path to the folder where niftii images are stored. Example: ~/<your_dataset>/data_processed',
     )
     parser.add_argument(
-        '-path-out',
+        '-path-label',
         metavar="<folder>",
         help=
-        "R|Path to the output folder where the corrected labels will be saved. "
-        "Default: './' (current directory). "
-        "Note: The path provided within this flag will be combined with the path provided within the "
-        "'-path-derivatives' flag. ",
-        default='./'
+        "Path to the folder where labels are stored. Example: ~/<your_dataset>/data_processed/derivatives/labels"
+        "\nDefault: -path-img will be used",
+        default=''
     )
     parser.add_argument(
-        '-path-derivatives',
+        '-path-out',
         metavar="<folder>",
         help=
-        "R|Path to the BIDS-compliant 'derivatives' folder where the corrected labels will be saved. "
-        "Default: 'derivatives/labels'. "
-        "Note: if the provided folder (e.g., 'derivatives/labels') does not already exist, it will be created. "
-        "Note: if the segmentation or label files already exist and you would like to correct them, provide path to "
-        "them within this flag.",
-        default=os.path.join('derivatives', 'labels')
+        "Path to the folder where corrected labels will be stored. Example: ~/<your_dataset>/data_processed/derivatives/labels"
+        "\nIf not specified: '-path-img + derivatives/labels' will be used. "
```

Comment:
If accepted, also other parts of the script will have to be modified.

```suggestion
        "\nIf not specified: '-path-label + derivatives/labels' will be used. "
```